packages/venia-concept/src/components/MiniCart/product.js

```diff
@@ -48,19 +50,26 @@ class Product extends Component {
 
     get options() {
         const { classes, item } = this.props;
+        const options = item.options ? JSON.parse(item.options) : [];
 
-        return item.options && item.options.length > 0 ? (
+        return options && options.length > 0 ? (
             <dl className={this.props.classes.options}>
```

Comment:
I know you didn't change this line but `classes` has already been destructured out of `this.props` and can be used directly here.

Reply:
good catch!